src/fragments/lib-v1/analytics/ios/record.mdx

```diff
@@ -42,6 +42,47 @@ Events have default configuration to flush out to the network every 60 seconds.
 
```

Comment:
Note for configuration sample above: need to verify whether the `autoFlushEventsInterval` is seconds or milliseconds. iOS sample shows seconds, Android sample shows milliseconds. That may be accurate, but wanted to double check

Reply:
I know Android using milliseconds is correct -- on the other hand, for Android v2 we [discovered](https://github.com/aws-amplify/docs/pull/4723/commits/7b5d20c4cb6390865962d894d4aae8dc8072151f#diff-4d5e7711c1d186d4081deab804db712043bc01e4ee21a8319f2a2eb606ac156c) we were reading the interval from inside the `pinpointAnalytics` block instead of just the `awsPinpointAnalyticsPlugin` block.  I can confirm if that is also the case in sdk/v1, but can iOS make sure that the config location is correct?

Reply:
@ruisebas will you be able to verify the iOS setting?

Reply:
This is not correct on iOS, the `autoFlushEventsInterval` option is read from inside the `awsPinpointAnalyticsPlugin` block as documented.
Also I can confirm iOS uses seconds.